cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -806,6 +806,14 @@
   </property>
 
   <property>
+    <name>master.collect.containers.log</name>
+    <value>true</value>
+    <description>
+      Collection of all containers log into master process log.
```

Comment:
I would suggest something like "Determines if container logs are streamed back to the cdap-master process log"